Solutions/Cloud Identity Threat Protection Essentials/Analytic Rules/MFADisable.yaml

```diff
@@ -38,15 +38,17 @@ query: |
  | summarize StartTimeUtc = min(TimeGenerated), EndTimeUtc = max(TimeGenerated) by User = TargetUser, Source = EventSource , Operation = EventName , TenantorInstance_Detail = InstanceProfileName, IPAddress = SourceIpAddress
```

Comment:
Instaed of using parsejson , so many times for same variable , can we go by doucmentation of parse_json 
use it like this 

 |extend d = parse_json(RequestParameters)
 | extend InstanceProfileName = tostring(d.InstanceProfileName)
 | extend TargetUser = tostring(d.userName)

Reply:
If it feels ok , can we do the same for other parse_json used in same query